rclcpp/include/rclcpp/executor.hpp

```diff
@@ -368,6 +374,10 @@ class Executor
 private:
   RCLCPP_DISABLE_COPY(Executor)
 
+  RCLCPP_PUBLIC
+  void
+  spin_once_impl(std::chrono::nanoseconds timeout);
```

Comment:
@ivanpauno I'd think the compiler needs no runtime tricks to use this method and so it _should_ be ABI compatible, but why the public visibility?

Reply:
> @ivanpauno I'd think the compiler needs no runtime tricks to use this method and so it _should_ be ABI compatible, but why the public visibility?

In the original PR, this method was protected, and not private.
I forgot changing the RCLCPP_PUBLIC macro to RCLCPP_LOCAL (though the change isn't completely necessary).

Reply:
Fixed